src/shared/components/Carousel/Carousel.style.ts

```diff
@@ -75,3 +49,37 @@ export const GliderContainer = styled.div<HasPadding>`
 export const Track = styled.div`
   align-items: flex-start;
 `
+
+export const Dots = styled.div`
+  margin-top: ${sizes(13)};
+
+  .glider-dot {
+    background-color: transparent;
+    width: ${sizes(10)};
+    border-radius: 0;
+    padding: ${sizes(1)};
+    margin: 0;
+
+    &::after {
```

Comment:
@drillprop I thought it might be more elegant than linear gradient. WDYT?

Reply:
yeah, looks good 👍🏻